### ISSUE TITLE
Si hay un input en un formulario dentro de  <article class="list scrollable">, no deja editarlo

### ISSUE THREAD

**Issue by kurnajata** [tapquo/Lungo.js#46]
Hola Javi,

Espero haberlo hecho bien. En la aplicación Kitchen-sink, en Forms-Element > Inputs he añadido muchos controles "input".Luego he añadido la clase "scrollable" al artículo  y ya no me deja editar los controles "input".

```
<article id="form-input" class="list scrollable">
    <div class="container indented">
        <ul class="form">
            ...
```

Saludos,

**Comment by campsjos** [comment]
I have the same problem using `lng.View.Scroll.create('container');` where #container is an article with select/option tags.

**Comment by christianarth** [comment]
Thats unfortunately an issue in iScroll:
http://code.google.com/p/iscroll-js/issues/detail?id=17

**Comment by soyjavi** [comment]
I'll check this solution:

```
            onBeforeScrollStart: function (e) {
                    var target = e.target;
                    while (target.nodeType != 1) target = target.parentNode;

                    if (target.tagName != 'SELECT' && target.tagName != 'INPUT' && target.tagName != 'TEXTAREA')
                            e.preventDefault();
            }
```

**Comment by campsjos** [comment]
it works!
Tested only on Chrome and Android 4.0 emulator

For who needs a fast solution:
In lungo-1.0.4.packed.js (line 70), replace:
`onBeforeScrollStart:function(u){u.preventDefault()}`
with
`onBeforeScrollStart:function(e){var target = e.target;while (target.nodeType != 1) target = target.parentNode;if (target.tagName != 'SELECT' && target.tagName != 'INPUT' && target.tagName != 'TEXTAREA'){e.preventDefault();}}`

**Comment by soyjavi** [comment]
Thanks guy!!! Release 1.1 is coming :) (and will be much more powerful and with less kB)

**Comment by campsjos** [comment]
Less code :)
Thx to: http://code.google.com/p/iscroll-js/issues/detail?id=17#c38

```
onBeforeScrollStart: function (e) {
    var target = e.target;
    while (target.nodeType != 1){
        target = target.parentNode;
    } 
    if (!target.form){
        e.preventDefault();
    }
} 
```

**Comment by christianarth** [comment]
I think the better solution is from soyjavi. It works even if no form element was used as wrapper.

**Comment by campsjos** [comment]
right, usually we wouldn't use `<form>` tag in Lungo :)

**Comment by soyjavi** [comment]
FIXED in Release 1.1

https://github.com/TapQuo/Lungo.js/commit/0a4142542093c2facf60389cd1b9ef14f5fbc685
